### ISSUE TITLE
serialized.indexOf is not a function console error

### ISSUE THREAD

**Issue by daraclare** [coopermaruyama/tableau-react#12]
Hi, thanks for the library! 
Issue: Using `React 16.6` and `Next 7`, I'm getting this error in the console: 
```
Uncaught TypeError: serialized.indexOf is not a function
    at Function.tab__ApiCommand$parse [as parse] (tableau-2.0.0.js:731)
    at tab__crossDomainMessageRouter$_handleCrossDomainMessage [as _handleCrossDomainMessage] (tableau-2.0.0.js:1025)
    at tab__crossDomainMessageRouter.<anonymous> (tableau-2.0.0.js:1017)
    at delegate (tableau-2.0.0.js:181)
```
It looks like `serialized` is an object, not a string or an array, so `indexOf` is not a valid function:
```
serialized: Object
reactBuildType: "development"
source: "react-devtools-detector"
```
Would updating to `tableau-2.2.2.js` resolve this? Or is this a problem when in development mode only?

Thanks in advance!

**Comment by Hartzi** [comment]
Any update to this?

**Comment by coopermaruyama** [comment]
Hi, this seems to be an issue in the child dependency of tableau-api which uses `tablueau-2.2.2.js` 

I no longer maintain this package, but it should be relatively easy to just add the new version into this repo and remove the child dependency. Feel free to open a PR

**Comment by kinly-wmusial** [comment]
Hi, PR seems good to me, hopefully we can merge it soon.

**Comment by glaucopater** [comment]
Hi, is there any update about the merge? Thx!

**Comment by coopermaruyama** [comment]
Sorry for the delay. I checked it, it was not wrapped correctly. I just published a new version using 2.3.0 which can be installed using `npm install tableau-react@beta`
